.github/actions/setup-build/action.yml

```diff
@@ -65,10 +70,11 @@ runs:
       shell: bash
 
     # Shorten binary names as a workaround for filepath length limits in Windows,
```

Comment:
And would simplify ci, and there would not be problems with long paths in windows, the name would be consistent with ghcup, etc, etc
I knoe it is not constructive but i cant resist to remember i suggested hls as executable name when all this started (like the executable for haskell-ide-engine was hie)
But now it would break the actual usage and who knows how many scripts and configurations out there so maybe it is too late.
If we decide to do the change it should have to be done very very carefully and gradually, supporting both names in the meanwhile 😓

Reply:
A simple symlink should resolve most of the script/config issues no?

Reply:
Yeah I don't know all the gory details with windows as I don't use it myself. Then the downstream itself I'm not sure how ghcup (?) would handle the symlinks and if that itself breaks anything? Seems like something to push down the road.

Reply:
> Then the downstream itself I'm not sure how ghcup (?) would handle the symlinks and if that itself breaks anything?

Using tiny executable wrappers called shims